src/Impostor.Api/Events/Game/IGameHostChangedEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Impostor.Api.Net;
+
+namespace Impostor.Api.Events
+{
+    public interface IGameHostChangedEvent : IGameEvent
+    {
+        IClientPlayer Host { get; }
+
+        IClientPlayer OldHost { get; }
```

Comment:
`PreviousHost` sounds better doesn't it?
Also move it above `NewHost` (remember about constructor parameter order)